src/test/java/org/cactoos/list/ImmutableTest.java

```diff
@@ -44,16 +43,66 @@
 @SuppressWarnings({"PMD.TooManyMethods", "PMD.AvoidDuplicateLiterals"})
 public class ImmutableTest {
 
-    @Test
-    public void innerListMustNotBeChanged() {
-        final List<String> strings = new ArrayList<>(Arrays.asList("a", "b", "c"));
-        final List<String> immutable = new Immutable<>(strings);
-        final int original = immutable.size();
-        strings.add("d");
-        new Assertion<>(
-            "inner list must not be changed",
-            original,
-            new IsEqual<>(immutable.size())
+    @Test(expected = UnsupportedOperationException.class)
```

Comment:
@victornoel How about using ` Throws` here, like in `subListReturnsListIteratorWithUnsupportedSet`?